update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_LOCATION="https://archive.raspbian.org/"
+RASPBIAN_ARCHIVE_KEY_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_LOCATION}${RASPBIAN_ARCHIVE_KEY_NAME}"
```

Comment:
The purpose of `LOCATION` is to point to the directory where the file is stored and thus the trailing / is important to denote that, otherwise it could give the impression it points to a file (`https://archive.raspberrypi.org/debian`).